gems/fog-dragonfly/CVE-2013-5671.yml

```diff
@@ -2,15 +2,15 @@
 gem: fog-dragonfly
 cve: 2013-5671
 osvdb: 96798
+ghsa: qrgf-jqqm-x7xv
 url: https://nvd.nist.gov/vuln/detail/CVE-2013-5671
 title: fog-dragonfly Gem for Ruby imagemagickutils.rb Remote Command Execution
 date: 2013-09-03
 description: |
   fog-dragonfly Gem for Ruby contains a flaw that is due to the program
   failing to properly sanitize input passed via the imagemagickutils.rb script.
   This may allow a remote attacker to execute arbitrary commands.
-
```

Comment:
I think the newline helps add emphasis. Maybe add a "Note:" to the last line?

Reply:
Add newline back